src/config.rs

```diff
@@ -744,6 +749,22 @@ enum CommandWrapper {
     },
 }
 
+impl CommandWrapper {
+    pub fn program(&self) -> &String {
+        match *self {
+            CommandWrapper::Just(ref program) => program,
+            CommandWrapper::WithArgs { ref program, .. } => program,
```

Comment:
you deref self and then `ref` it again, it seems not needed, just match on self directly, you'll get the reference:
```rust
match self {
     CommandWrapper::Just(program) => program,
}
```
(btw, `ref` is not "recommended" anymore since [1.26](https://github.com/rust-lang/rust/pull/49394))

Reply:
Old habits die hard :D. Thanks for the review!